ingress/controllers/gce/controller/utils.go

```diff
@@ -34,7 +34,10 @@ import (
 	"github.com/golang/glog"
 )
 
-const allowHTTPKey = "kubernetes.io/ingress.allowHTTP"
+const (
+	allowHTTPKey    = "kubernetes.io/ingress.allow-http"
```

Comment:
For future - these should be `ingress.kubernetes.io/allow-http` - it's documented somewhere that I am sure you didn't see :)